utils/cq/requirements.txt

```diff
@@ -1,14 +1,15 @@
-# Packages needed to pylint code.  Most of these are needed for test rather than build.
+# Packages needed to check code.  These are needed for linting rather than build.
 # 94.0 is reporting a couple of errors so stick on 93.
 avocado-framework<94
 avocado-framework-plugin-result-html<94
 avocado-framework-plugin-varianter-yaml-to-mux<94
 clustershell
+paramiko
+pyenchant
 ## flake8 6 removed --diff option which breaks flake precommit hook.
 ## https://github.com/pycqa/flake8/issues/1389 https://github.com/PyCQA/flake8/pull/1720
 flake8<6.0.0
-isort
-paramiko
-pyenchant
-pylint
-yamllint
+isort==5.12
+pylint==3.0.2
+yamllint==1.32.0
+codespell==2.2.6
```

Comment:
Just pointing out this requires a newer python. My distro default was python3.6, which did not have isort 5.12 available. After upgrading to python3.9, this works. I talked with @ashleypittman and really we need better dev setup documentation

Reply:
I agree, but it's no reason for CI and power users to have access to the latest versions of the tools.